functions/code/EventHubCopy/Configure-Function.ps1

```diff
@@ -0,0 +1,41 @@
+[CmdletBinding()]
+param (
+    # Name of the Function App
+    [Parameter(Mandatory)]
+    [String]
+    $FunctionAppName,
+    # Name of the Resource Group
+    [Parameter(Mandatory)]
+    [String]
+    $ResourceGroupName,
+    # Source Event Hubs Namespace
+    [Parameter(Mandatory)]
+    [String]
+    $SourceNamespacename,
+    # Name of the source Event Hub
+    [Parameter(Mandatory)]
+    [String]
+    $SourceEventHubName,
+    # Target Event Hubs Namespace
+    [Parameter(Mandatory)]
+    [String]
+    $TargetNamespacename,
+    # Name of the target Event Hub
+    [Parameter(Mandatory)]
+    [String]
+    $TargetEventHubName
+)
+
+$null = az eventhubs eventhub authorization-rule create --resource-group $ResourceGroupName --namespace-name $SourceNamespacename --eventhub-name $SourceEventHubName --name replication-listen --rights listen
```

Comment:
It's a bit odd to use the Azure CLI inside of a PowerShell script rather than using the PowerShell cmdlets.   What was behind that decision?

Reply:
I chose to use the Azure CLI since the README's in this sample used it. I think it would be easy to change to the cmdlets. Since it worked and ran successfully, I didn't realize that it was odd, but looking back at it I see how it would make more sense to use the cmdlets.

Reply:
I'm all for leaving it; definitely not worth the efforts to re-engineer things.   

Besides, if the README is already pulling in the CLI, then it's a known dependency.   We should just be sure to mention that they need PowerShell in the README as well - assuming they're already calling out prerequisites.   Something like our standard blurb would work:

> PowerShell Core version 7.0 or greater is needed.  This version is available cross-platform and can be installed from the [PowerShell Core repository](https://github.com/PowerShell/PowerShell/blob/master/README.md).

Reply:
Also, the ps1 files are what I needed to use to run all the tests, I could just move them into the test folder?

Reply:
The test originally had been calling the deploy-function, deploy-resources, etc ps1 files in the location they are now, but they were all missing, so I was just re-adding them in.